eng/common/scripts/Update-ChangeLog.ps1

```diff
@@ -12,18 +12,21 @@ param (
   [String]$ServiceDirectory,
   [Parameter(Mandatory = $true)]
   [String]$PackageName,
-  [boolean]$Unreleased=$True,
-  [boolean]$ReplaceLatestEntry = $False,
+  [String]$Unreleased=$True,
+  [String]$ReplaceLatestEntryTitle = $False,
   [String]$ReleaseDate
 )
 
-if ($ReleaseDate -and ($Unreleased -eq $True)) {
+[Boolean]$Unreleased = [System.Convert]::ToBoolean($Unreleased)
+[Boolean]$ReplaceLatestEntryTitle = [System.Convert]::ToBoolean($ReplaceLatestEntryTitle)
+
+. (Join-Path $PSScriptRoot common.ps1)
+
+if ($ReleaseDate -and $Unreleased) {
     LogError "Do not pass 'ReleaseDate' arguement when 'Unreleased' is true"
     exit 1
 }
 
-. (Join-Path $PSScriptRoot common.ps1)
```

Comment:
Don't you need this for some of the functions you use below?

Reply:
Yes it is needed but he only moved it up a few lines.